src/browser/components/MainPage.jsx

```diff
@@ -281,7 +294,8 @@ export default class MainPage extends React.Component {
     this.handleOnTeamFocused(newKey);
   }
 
-  handleBadgeChange = (index, sessionExpired, unreadCount, mentionCount, isUnread, isMentioned) => {
```

Comment:
Component methods should either be all arrow function variables or all functional declarative and bound in constuctor